changelogs/fragments/85_helm_plugin.yaml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
```

Comment:
```suggestion
breaking_changes:
```

Reply:
Unless there's some sort of weird caching issue going on with Github, it looks like this didn't get changed?